### PR TITLE
[CI] Test post comment

### PR DESCRIPTION
**Pull request by zjjlivein**
#### Before submitting

- [ ] Lint code. If there are lint issues, please format the code first.

```shell
# Install and register `pre-commit` in the project folder
pip install pre-commit && pre-commit install

# Process previous code files separately
pre-commit run --file XXXX.py
```

- [ ] Add test cases into `tests` folder. If there are codecov issues, please add tests cases first.

### CODE REVIEW

**Comment by zjjlivein** [comment]
## 🤖 PaddleFormers Bot Log Analysis

> Run [`#26015275932`](https://github.com/PaddlePaddle/PaddleFormers/actions/runs/26015275932) · Attempt `1`

## 日志分析报告

| 流水线名称 | 问题标签 | 修复建议 |
|------------|----------|----------|
| CI_ILUVATAR | 单测存在 Bug（LossNan） | `test_ernie_21b_sft_training` 训练第 1 步后 Loss 变 nan，需排查本次 PR 是否引入了数值精度问题，重点检查数据类型隐式转换逻辑 |

---

**失败的测试 case**:
```
scripts/iluvatar_ci/test_ernie_21b_sft.py::test_ernie_21b_sft_training
```

---

**根本原因分析**:

训练脚本 `paddleformers/cli/train/sft/workflow.py:750 → trainer.py:2568` 在 `_check_loss_valid` 检测到 Loss 为 `nan` 并主动抛出异常。

关键信息：
- **global_step=1** 时 loss=11.34，属正常初始值，但 `global_norm=30.47`（异常偏高）
- **step=2 时 loss 变为 nan**，说明第二步梯度更新后数值溢出
- 日志出现两类数据类型警告：
  - `multiply_fwd_func.cc:86`: `got different data type, run type promotion`（乘法混合精度隐患）
  - `dygraph_functions.cc:125086`: `Got different data type, run type promotion`（动态图算子隐式类型提升）
- 这些警告说明存在 **fp16/bf16 与 fp32 混用**，可能导致中间计算溢出，进而 Loss 变 nan

---

**修复建议**:

1. **检查本次 PR 改动**中是否有影响 ERNIE-21B SFT 数值精度的代码（如梯度计算、混合精度 scaler、MoEGate 相关逻辑），重点关注 `trainer.py` 和 `sft/workflow.py` 的修改
2. **排查数据类型混用**：`dygraph_functions.cc` 和 `multiply_fwd_func.cc` 均触发了隐式类型提升，需检查 bf16/fp16 tensor 与 fp32 参数混合运算的场景，确保显式 cast 到统一类型
3. **global_norm=30.47 偏高**：检查是否关闭了 gradient clipping 或 clip_norm 阈值配置异常，必要时降低 learning_rate 或启用 loss scaling
4. **建议先 merge develop 最新代码**后重跑，确认是否为本 PR 引入的问题

---
<sub>🔄 每次 Re-run 后自动更新</sub>
